arbnode/node.go

```diff
@@ -1343,7 +1373,12 @@ func CreateNodeExecutionClient(
 	parentChainID *big.Int,
 	blobReader daprovider.BlobReader,
 	latestWasmModuleRoot common.Hash,
+	useRPC bool,
 ) (*Node, error) {
+	if useRPC {
```

Comment:
why give a boolean that means "ignore executionClient" and not just give it the correct execution client?